Benchmarks/TestGeneratorControlFiles/BoneAndMarrow.control

```diff
@@ -0,0 +1,98 @@
+%
+% ------------------------------------------------------------------
+% Control file for two materials embedded in a background material.
+% ------------------------------------------------------------------
+%
+% -------------
+% Control Block
+% -------------
+%
+\begin{CONTROL_INPUT}
+%
+%  Plot and stats file names can be "none" if no output
+%  is desired. 
+%
+   \begin{RUN_PARAMETERS}
+      mesh file name   = Benchmarks/MeshFiles/Tests/BoneAndMarrow.mesh
+      plot file name   = Benchmarks/PlotFiles/Tests/BoneAndMarrow.tec
+      stats file name  = Benchmarks/StatsFiles/Tests/BoneAndMarrow.txt
+      test file name   = Benchmarks/BenchmarkData/BoneAndMarrow.txt
+      mesh file format = ISM-MM
+      polynomial order = 5
+      plot file format = skeleton
+   \end{RUN_PARAMETERS}
+
+   \begin{MESH_PARAMETERS}
+      element type = quad
+   \end{MESH_PARAMETERS}
+
+   \begin{BACKGROUND_GRID}
+      background grid size = [1.0,1.0,0.0]
+   \end{BACKGROUND_GRID}
+%
+%  Spring smoother parameters are pretty standard after lots
+%  of trials.
+%
+   \begin{SPRING_SMOOTHER}
+      smoothing type       = LinearAndCrossBarSpring
+      number of iterations = 25
+   \end{SPRING_SMOOTHER}
+
+\end{CONTROL_INPUT}
+%
+% -----------
+% Model Block
+% -----------
+%
+\begin{MODEL}
+   
+   \begin{MATERIALS}
+      background Material name = Muscle
```

Comment:
Now using "background material name" since it is more descriptive. Fixed in https://github.com/trixi-framework/HOHQMesh/commit/c69b035d6b3a1a9c9bb4eaba772dc79a4748e98f